src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/CaseManagementForCaseWorkerService.java

```diff
@@ -318,4 +335,36 @@ private void sendUpdateSingleCaseECC(String authToken, CaseDetails currentCaseDe
         }
     }
 
+    public static void populateConciliationTrack(CaseData caseData) {
```

Comment:
CaseManagementForCaseWorkerService is a Spring bean. You should never use a static method in this case as we are using dependency injection. Clients will always inject the class as a dependency - they would never call a static method